updatecli/updatecli.d/charts/mirrorbits.yaml

```
@@ -0,0 +1,42 @@
+---
+name: Bump `mirrorbits-cli` version
```

```suggestion
name: Bump `mirrorbits` helm chart version
```

Blocker: need to be up to date